akka-docs/src/main/paradox/persistence.md

```diff
@@ -697,6 +708,12 @@ where resilience is important so that if a node crashes the persistent actors ar
 resume operations @ref:[Cluster Sharding](cluster-sharding.md) is an excellent fit to spread persistent actors over a 
 cluster and address them by id.
 
+Akka Persistence is based on the single-writer principle which means that it is important to avoid running the same
```

Comment:
Maybe: Akka Persistence is based on the single-writer principle. Only one entity instance should be writing to a particular data store at one time. If multiple entity instances were to persist data at the same time, the events would be interleaved and could not be interpreted correctly on replay. Cluster Sharding gives you the ability to run multiple instances of an entity within a data center and Multi-DC Persistence solves this issue across data centers by supporting active-active persistent entities. ???

Reply:
There are a few things that are not technically correct with that rewrite. Is there something specific that is unclear or wrong with the original text?

Reply:
Yes, the description was unclear to me. That's probably why my rewrite is wrong. :-(

Let me ask my questions to you in an email. Then, maybe I can make a better suggestion.